src/fairseq2/recipes/lm/_online_finetune/_rewards.py

```diff
@@ -264,16 +279,29 @@ def process_rollouts(
                     rollout_output.text, i_reference_answer
                 )
                 rollouts_rewards.append(predicted_reward)
+                vllm_input = self.wrap_text("dummy prompt", "dummy rollout")
+                vllm_inputs.append(vllm_input)
+
             batch_text.append(rollouts_text)
             batch_tokens.append(rollouts_tokens)
             batch_rewards.append(rollouts_rewards)
 
+        # dummy vllm call for syncronization # FIXME
+        # if self.vllm_math_reward_model is not None:
+        log.info("MathVerifyVerifir generate_rewards()")
+        dummy_rewards = generate_rewards(
```

Comment:
if i check for `if self.vllm_math_reward_model is not None` which it will be None for non combination runs, then i get an nccl error. not sure what's happening there